tests/e2e/testfiles/storage_csm_powermax.yaml

```diff
@@ -53,6 +53,17 @@ spec:
       # Default value: None
       imagePullPolicy: Always
       envs:
+        # X_CSI_MANAGED_ARRAYS: Serial ID of the arrays that will be used for provisioning
+        # Default value: None
+        # Examples: "000000000001", "000000000002"
+        - name: X_CSI_MANAGED_ARRAYS
+          value: "000000000000,000000000001"
```

Comment:
for the e2e tests, value needs to be
REPLACE_ENDPOINT. 
See tests/e2e/testfiles/powermax-templates/powermax-array-config.yaml 
for a list of the values needed for the fields being moved over form the configmap

Reply:
Hi @JacobGros ,
I see that other YAML files for Pmax or the Other drivers do not use this convention for providing the values.
Why is this an exception?? This is present in the testfiles directory, not inside the testfiles/powermax-templates folder

Reply:
discussed on call, latest commit fixes this concern